api/v1beta1/tags.go

```diff
@@ -74,10 +75,13 @@ func (t Tags) Merge(other Tags) {
 	}
 }
 
-// Validate checks if tags are valid for the AWS API. Keys must have at
-// least 1 character and max 128. Values must be max 256 characters long.
+// Validate checks if tags are valid for the AWS API/Resources.
```

Comment:
Keeping validations we have in this method as comment is useful as we only have checks for a subset of them.

Reply:
okay will add comments describing the checks